profiles/commands.py

```diff
@@ -15,6 +20,34 @@ def _update_name_from_orcid_record(profile: Profile, orcid_record: dict) -> None
                             '{}, {}'.format(family_name, given_name))
 
 
+def _update_affiliations_from_orcid_record(profile: Profile, orcid_record: dict) -> None:
```

Comment:
https://github.com/elifesciences/orcid-dummy/pull/4 provides data for this in end2end